### PR TITLE
pokerstars.rb: add region support

### PR DESCRIPTION
**Pull request by vitorgalvao**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

---

@reitermarkus Requesting some help here. I feel like this could be more elegant. There are a few things to consider:

+ PokerStars will only work if you have the correct version for your region. There used to be an `EU` version, but since now in Portugal it is `PT`, I’m guessing they have one for each country specifically.
  + Because of that, there really shouldn’t be a “default” version. Would it be possible to make it so that a user *has to* specify a region, else it would fail to install?
+ PokerStars redirects you like crazy. They seriously do not by any means want you visiting the the pages for the other regions, which makes verifying any region but yours a pain (which is why I only added one language).

### CODE REVIEW

**Comment by Numbermaniac** [comment]
Maybe that needs to be done for the homepage too? Looks like https://www.pokerstars.pt/ has its own homepage.

**Comment by vitorgalvao** [comment]
@Numbermaniac I had intentionally not done that to the homepage since it redirects you anyway. But it actually makes more sense since it makes the comment no longer necessary.

I’ve also changes `us` to `en`. Since it’s the default, we can refine it later if necessary.

Also going to add the EU version, seems like that may still be around.

**Comment by reitermarkus** [comment]
Seems like `pokerstars.pt` is an exception, as `pokerstars.com` redirects me to `pokerstars.eu` and there specifically is no option to switch to Portoguese, but a bunch of other languages.

All of these link to `pokerstars.eu/xx/`, except for:

- Latvian: `pokerstars1.eu` (actually links to `pokerstars.eu/lt/` but redirects)
- Magyar `pokerstars-cc.eu`
- Romanian `pokerstars.ro`

**Comment by vitorgalvao** [comment]
@reitermarkus Even if I go to the Romanian one, when trying to download I still get the PT software.

In terms of cask, this is easy to fix: either we scale it back further — `homepage "https://www.pokerstars.#{language[0]}"` becomes `homepage "https://www.pokerstars#{language[0]}"` (the `.` itself is removed and added in `language` — or we go back to `homepage 'https://www.pokerstars.com'` and let the redirect happen on their own.

I don’t have a strong preference. I wouldn’t mind `homepage "https://www.pokerstars.#{language[0]}"` for now and see how it goes. It would be fairly trivial to change to `.com` always if that ends up being too much.

What does matter is the termination in the software’s name. We can only add the ones we’re certain about. Which termination do you have, from which location?

**Comment by reitermarkus** [comment]
Downloading from Austria I get `PokerStarsEU.app` (via `https://www.pokerstars.eu/de/client/download/`, although `https://www.pokerstars.eu/client/download/` is the same).

File: Casks/pokerstars.rb [hunk 1]
```diff
@@ -2,17 +2,30 @@
   version :latest
   sha256 :no_check
 
-  url 'https://www.pokerstars.com/PokerStars.app.zip'
+  language 'en', default: true do
+    %w[com]
+  end
+
+  language 'pt' do
```
Comment (reitermarkus) [inline Casks/pokerstars.rb]: This should probably be `'PT'`, since this is region-specific, not language-specific.

File: Casks/pokerstars.rb [hunk 2]
```diff
@@ -2,17 +2,30 @@
...
+    %w[pt PT]
+  end
+
+  language 'eu' do
```
Comment (reitermarkus) [inline Casks/pokerstars.rb]: Also `eu` is not a valid language code, so this should be similar to https://github.com/caskroom/homebrew-cask/blob/9a91f5ae67979652d72943e76d9f5c368f7058d6/Casks/world-of-tanks.rb#L5.

**Comment by reitermarkus** [comment]
Also I just found this: https://www.pokerstars.eu/languages/

**Comment by vitorgalvao** [comment]
Fixed with the languages we’re sure of.

> Also I just found this: https://www.pokerstars.eu/languages/

Can’t see it, it redirects me to the PT site once more (it also does so with help articles, it’s really annoying and user-hostile).

**Comment by adidalal** [comment]
screen shot 2016-12-04 at 5 12 54 pm

File: Casks/pokerstars.rb [hunk 3]
```diff
@@ -2,17 +2,34 @@
   version :latest
   sha256 :no_check
 
-  url 'https://www.pokerstars.com/PokerStars.app.zip'
+  language 'US', default: true do
+    %w[.com]
+  end
+
+  language 'AU' do
```
Comment (reitermarkus) [inline Casks/pokerstars.rb]: I assume you mean Austria, which is `AT`.

File: Casks/pokerstars.rb [hunk 4]
```diff
@@ -2,17 +2,34 @@
...
+    %w[.eu/de EU]
```
Comment (reitermarkus) [inline Casks/pokerstars.rb]: Just `.eu` is enough. I just rechecked and on the downloads page (`https://www.pokerstars.eu/de/poker/download/mac/`) it is actually linked without `/de/`.

File: Casks/pokerstars.rb [hunk 5]
```diff
@@ -2,17 +2,34 @@
...
+  end
+
+  language 'AT' do
+    %w[.eu EU]
+  end
+
+  language 'BE' do
+    %w[.be BE]
+  end
+
```
Comment (Numbermaniac) [inline Casks/pokerstars.rb]: think you left an accidental whitespace here :)

**Comment by Numbermaniac** [comment]
https://web.archive.org/web/20161205070028/https://www.pokerstars.com/languages/

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**